covid19-etl/etl/idph_vaccine.py

```diff
@@ -0,0 +1,268 @@
+        for county in counties:
+            county_covid_response = requests.get(
+                COUNTY_COVID_LINK_FORMAT.format(county),
+                headers={"content-type": "json"},
+            )
+            county_covid_data = json.loads(county_covid_response.text).get(
+                "CurrentVaccineAdministration"
+            )
+            county_demo_response = requests.get(
+                COUNTY_DEMO_LINK_FORMAT.format(county), headers={"content-type": "json"}
+            )
+            county_demo_data = json.loads(county_demo_response.text)
+
+            (
+                summary_location_submitter_id,
+                summary_clinical_submitter_id,
+            ) = self.get_location_and_clinical_submitter_id(county, self.date)
+            if county.lower() == "illinois":
+                state_summary_clinical_submitter_id = summary_clinical_submitter_id
+
+            for k in ["Age", "Race", "Gender"]:
+                data = county_demo_data.get(k)
+                for item in data:
+                    keys, props = self.parse_group_clinical_demographic(
+                        county_demo_mapping, item
+                    )
+                    group_demographics_submitter_id = (
+                        self.get_group_clinical_demographic_submitter_id(
+                            summary_clinical_submitter_id, keys
+                        )
+                    )
+                    props["submitter_id"] = group_demographics_submitter_id
+                    props["summary_clinicals"] = [
+                        {"submitter_id": summary_clinical_submitter_id}
+                    ]
+                    self.summary_group_demographic[
+                        group_demographics_submitter_id
+                    ] = props
+
+            summary_location = {
+                "country_region": self.country,
+                "submitter_id": summary_location_submitter_id,
+                "projects": [{"code": self.project_code}],
+                "province_state": self.state,
+            }
+            summary_clinical = {
+                "submitter_id": summary_clinical_submitter_id,
+                "date": self.date,
+                "summary_locations": [{"submitter_id": summary_location_submitter_id}],
+            }
+            for (key, value) in county_vaccine_mapping.items():
+                if key == "PctVaccinatedPopulation":
+                    summary_clinical[value] = int(county_covid_data.get(key) * 100)
+                elif key == "AdministeredCountRollAvg":
+                    summary_clinical[value] = int(county_covid_data.get(key))
+                else:
+                    summary_clinical[value] = county_covid_data.get(key)
+            # for (key, value) in county_demo_mapping.items():
+            #     summary_clinical[value] = county_demo_data.get(key)
+            for (key, value) in inventory_reported.items():
+                if county in self.counties_inventory:
```

Comment:
the `counties` list and the `self.counties_inventory` dict are created at the same time during `parse_list_of_counties()`. So i don't think you need `if county in self.counties_inventory`, it's always True, right?